### DISCUSSION TITLE
Communication

### DISCUSSION THREAD

**Discussion by ghost** [Beep6581/RawTherapee#6898] [General]
Hello to everyone,

I would like to have some sort of a chatting possibility to get in contact with developers of RawTherapee. I can't find any, except of slow communication channels here on GitHub. Most of the time, I have just small questions a whole post is not necessary for.

Is there any official or inofficial channel for developers of RawTherapee?

If there is non should we create one discord or whatever chat possibility?

Thank you in advance.
Greetings

**Comment by Beep6581** [comment]
Hey

If the questions are of a user support nature then please use the forum, else if they're related to development then use GitHub.

**Reply by ghost** [reply]
The questions are related to development. But for a lot of many small questions, the github communication seem to be  overloaded. I got a lot of small questions here and there for a lot of code parts.

Well, maybe in future anyone might create a new pragmatic communication channel for RawTherapee development.

Thanks
